apps/chat/src/store/share/share.epics.ts

```diff
@@ -270,6 +270,8 @@ const acceptInvitationEpic: AppEpic = (action$) =>
                 ShareActions.acceptShareInvitationSuccess({
                   acceptedId: ApiUtils.decodeApiUrl(acceptedIds[0].url),
                   isFolder: isFolderId(data.resources[0].url),
```

Comment:
maybe we should check if folder is a chat/prompt folder?

Reply:
I'm not sure if I understand you correctly, but I believe that this three flags should work good enough. Also I pushed one more commit and reused these isPromt and isConversation near to isFolder further down the code.